Core/clim-core/presentations/presentation-types.lisp

```diff
@@ -42,8 +42,21 @@
 (defvar *print-presentation-verbose* nil)
 
 (defmethod initialize-instance :after ((instance presentation-mixin) &key)
-  (unless (presentation-type instance)
```

Comment:
Typos in commit message
"presentation is created AND the object is not"
"signal the error, when we try TO present"

Reply:
fixed